libp2p/tools/async_service/trio_service.py

```diff
@@ -416,7 +427,12 @@ async def inner(self: ServiceAPI, *args: Any, **kwargs: Any) -> Any:
         async with trio.open_nursery() as nursery:
             # mypy's type hints for start_soon break with this invocation.
             nursery.start_soon(
-                _wait_api_fn, self, func, args, kwargs, send_channel  # type: ignore
+                _wait_api_fn,  # type: ignore
```

Comment:
This is a preexisting `type: ignore`, but can it be resolved in this PR? Fine if not.